### PR TITLE
test: add missing --color argument

### PR DESCRIPTION
**Pull request by mschoettle**
Adds support for running the tests of this plugin with flake8 v5

### CODE REVIEW

**Comment by mschoettle** [comment]
@awelzel: Please review

File: tests/test_formatter.py [hunk 1]
```diff
@@ -12,9 +12,10 @@
 class TestGitlabCodeClimateFormatter(unittest.TestCase):
 
     def setUp(self):
-        self.options = unittest.mock.Mock(["output_file", "tee"])
+        self.options = unittest.mock.Mock(["output_file", "tee", "color"])
         self.options.output_file = tempfile.mktemp(suffix=".json", dir=".")
         self.options.tee = False
+        self.options.color = 'auto'
```
Comment (awelzel) [inline tests/test_formatter.py:18]:
```suggestion
        self.options.color = "auto"
```

**Comment by mschoettle** [comment]
I realized that this makes this flake8 v5 specific. With flake8 v3 and v4 it fails due to an unrecognized argument:

> error: unrecognized arguments: --color

**Comment by awelzel** [comment]
> I realized that this makes this flake8 v5 specific. With flake8 v3 and v4 it fails due to an unrecognized argument:

This has been fixed up in master. CI is now also running with multiple flake8 versions.

**Pull request merged**